apps/docs/components/Navigation/Footer.tsx

```diff
@@ -44,11 +44,17 @@ const Footer = () => (
           <a className="text-xs text-scale-900">&copy; Supabase Inc</a>
         </Link>
         <span className="text-xs text-scale-900">—</span>
-        {secondaryLinks.map((item) => (
-          <Link href={item.url} key={item.url}>
-            <a className="text-xs text-scale-800 hover:underline">{item.title}</a>
-          </Link>
-        ))}
+        {secondaryLinks.map(({ component: Component, ...item }) =>
+          item.url ? (
+            <Link href={item.url} key={item.url}>
+              <a className="text-xs text-scale-800 hover:underline">{item.title}</a>
```

Comment:
text-foreground-lighter

Reply:
Ok, migrated every occurrence of scale colors in www and docs footers (borders, text etc) @MildTomato